wordpress-mobile/gutenberg-mobile
# Remove unnecessary <strong> tags from post title

This is a quick workaround to remove `<strong>` tags from post title. We can see those tags on Aztec editor.

![title-strong-tag2](https://user-images.githubusercontent.com/5032900/52470598-ad5f2900-2b9f-11e9-94fc-24ce9fbd73f3.gif)

Changes reside in  https://github.com/WordPress/gutenberg/pull/13763

**To Test**

Test both on WPAndroid and WPiOS
Main apps PRs: [wpandroid](https://github.com/wordpress-mobile/WordPress-Android/pull/9196), [wpios](https://github.com/wordpress-mobile/WordPress-iOS/pull/10969).

1. Test with a new post

- Open a new post 
- Type a title
- Switch to Classic editor
- Verify that you don't see `<strong>` tags

2. Test with an existing post

- Open an existing post with a title
- Switch to Classic editor
- Verify that you don't see `<strong>` tags
 
3. Test we don't affect other blocks

- Open a post and add a paragraph/heading block
- Select all and make bold
- Switch to classic editor and see boldness didn't got lost
